config/routes.rb

```diff
@@ -1,11 +1,13 @@
 Rails.application.routes.draw do
   devise_for :users, controllers: { sessions: "users/sessions" }
-
+  
   root "home#index"
-
+  
   get "stats", to: "stats#index"
   get "my_requests", to: "trip_join_requests#show_my_requests"
-
+  
+  resources :companions
```

Comment:
I would move this route under `user`; at the end they are companions of someone, isn't it?